### PR TITLE
fix: Properly document context params

### PR DESCRIPTION
**Pull request by jlacivita**

### CODE REVIEW

File: util/docs/macros/index.mjs [hunk 1]
```diff
@@ -1017,12 +963,12 @@ function generateJavaScriptExampleResult(example) {
 }
 
 function generateRPCExample(example, m, moduleJson = {}) {
-    if (m.tags && m.tags.filter(t => (t.name === 'property-subscribe')).length) {
-        return generatePropertyChangedRPCExample(example, m, moduleJson)
-    }
-    else if (m.tags && m.tags.filter(t => (t.name === 'property-set')).length) {
-        return generatePropertySetRPCExample(example, m, moduleJson.info.title)
-    }
+    // if (m.tags && m.tags.filter(t => (t.name === 'property-subscribe')).length) {
```
Comment (kpears201) [inline util/docs/macros/index.mjs:966]: How come these are commented out?

Comment (jlacivita) [reply util/docs/macros/index.mjs]: These examples are generated using the standard code now, removed the commented out code (and unused methods the comments referred to)

**Review by kpears201 — APPROVED** [review]

**Pull request merged**